deploy/helm/sumologic/values.yaml

```
@@ -2821,12 +2834,60 @@ otelcol:
                 from_attribute: k8s_container_name
               - action: delete
                 key: k8s_container_name
-          groupbyattrs:
+          attributes/remove_fluent_tag:
+            actions:
+              - action: delete
+                key: fluent.tag
+
+          groupbyattrs/fluent_tag:
+            keys:
+              - fluent.tag
+          groupbyattrs/containers:
             keys:
               - k8s.container.id
               - k8s.container.name
               - k8s.namespace.name
               - k8s.pod.name
+          groupbyattrs/systemd:
+            keys:
+              - _SYSTEMD_UNIT
+              - _HOSTNAME
+
+          filter/fluent_tag_otelcol_container:
+            logs:
+              exclude:
+                match_type: regexp
+                record_attributes:
+                  - key: fluent.tag
+                    value: "containers.var.log.containers.collection-sumologic-otelcol-logs.*"
```

not big change, but still

```suggestion
                    value: "containers\.var\.log\.containers\.collection-sumologic-otelcol-logs.*"
```